contribute/design/1.x/cstor_cr_schema/apis.md

```diff
@@ -0,0 +1,528 @@
+# Moving CSPC and CSPI Schema to v1
```

Comment:
can we move them inside `contribute/design/1.x/apis/cstor_pool.md` and later we can add the other apis under the same folder 

- contribute/design/1.x/apis/cstor_pool.md
- contribute/design/1.x/apis/cstor_volume.md
- contribute/design/1.x/apis/jiva_volume.md